apiserver/plane/api/views/base.py

```
@@ -86,7 +86,8 @@ def handle_exception(self, exc):
                 )
 
             if isinstance(e, ValidationError):
-                import traceback; traceback.print_exc()
+                import traceback
```

Pylint (3.3.7)

[convention] Import outside toplevel (traceback)

(C0415)